README.md

```diff
@@ -10,11 +10,16 @@
 To install the project dependencies, run the following command:
 
 ```bash
-poetry install
+$ poetry install
+```
+
+### Run test suite
+```bash
+$ ./run_tests.sh
```

Comment:
También aumentaría de una vez en el README como correr es decir hacer el `poetry run python src/__main__`

Reply:
I would actually advocate for adding a `you_owe_me.sh` bash script instead than adding it to the README. It makes the experience much smoother and better for users I think.

Reply:
@Quezma @NicolasWebDev ya se agrego el script